### ISSUE TITLE
no check on dynamic imports

### ISSUE THREAD

**Issue by manfredsteyer** [softarc-consulting/sheriff#1]
Let's assume we bypass the index.ts like this:

```
import { CheckinService } from '../data/checkin.service';
```

This seems to be fine as long as index.ts also export the accessed module:

```
export * from './checkin.service';
```

However, when accessing something else, we get the expected lining error:

```
import { HiddenService } from '../data/hidden.service';
```

# Expected Behavior

I'm not sure if the current behavior is the intended one. At least, it can be confusing.

**Comment by rainerhahnekamp** [comment]
Nope, that is strange. It is deep-import and is forbidden.

**Comment by rainerhahnekamp** [comment]
The problem was narrowed down. The reason is that dynamic imports are not covered by Sheriff yet.

**Comment by rainerhahnekamp** [comment]
That was fixed in #9  and is available since version 0.10.0

**Comment by rainerhahnekamp** [comment]
Closing it since it is already working for some time
